client/state/sites/plans/reducer.js

```diff
@@ -1,39 +1,86 @@
 /**
+ * External dependencies
+ */
+import omit from 'lodash/object/omit';
+
+/**
  * Internal dependencies
  */
 import {
-	FETCH_SITE_PLANS,
-	FETCH_SITE_PLANS_COMPLETED,
-	REMOVE_SITE_PLANS
-} from './action-types';
-import omit from 'lodash/object/omit';
+	SITE_PLANS_FETCH,
+	SITE_PLANS_FETCH_COMPLETED,
+	SITE_PLANS_FETCH_FAILED,
+	SITE_PLANS_REMOVE,
+	SITE_PLANS_TRIAL_CANCEL,
+	SITE_PLANS_TRIAL_CANCEL_COMPLETED,
+	SITE_PLANS_TRIAL_CANCEL_FAILED
+} from 'state/action-types';
 
 export const initialSiteState = {
+	data: null,
 	error: null,
 	hasLoadedFromServer: false,
 	isFetching: false,
-	data: null
+	isUpdating: false
```

Comment:
I agree that `hasLoadedFromServer` is not the best name. I think `isStale` could be a good alternative since it's short and it follows the `isSomething` syntax. I refrained from updating this pull request to address this specific change though since it is already quite long.